src/payment-information/entities/user-payment-information.entity.ts

```diff
@@ -0,0 +1,25 @@
+import { ObjectType, Field } from '@nestjs/graphql';
+import { GraphQLJSONObject } from 'graphql-type-json';
+import { UUID } from 'src/shared/scalars/CustomUuidScalar';
+import { PaymentMethod } from 'src/payment-method/payment-method.enum';
+
+// to remove user information from all non-employee queries
+@ObjectType({ description: 'The users view of a stored payment information' })
+export class UserPaymentInformation {
```

Comment:
also if you really want to hide it, couldn't you just limit access in the user field resolver?

Reply:
I think, we should not return the user entity to the end user.
There is no reason for them to have it and it would expose internals.
However limiting access to the user field resolver might be a better solution.

Reply:
in the other services, our solution was to cause an error if you access a field which is forbidden to you